UM/OutputDevice/OutputDeviceManager.py

```diff
@@ -14,13 +14,20 @@
     from UM.OutputDevice.OutputDevicePlugin import OutputDevicePlugin
 
 
-# Used internally to determine plugins capable of 'manual' addition of devices, see also [add|remove]ManualDevice below.
 class ManualDeviceAdditionAttempt(Enum):
-    NO = 0,        # The plugin can't add a device 'manually' (or at least not with the given parameters).
-    POSSIBLE = 1,  # The plugin will try to add the (specified) device 'manually', unless another plugin has priority.
-    PRIORITY = 2   # The plugin has determined by the specified parameters that it's responsible for adding this device
-                   #     and thus has priority. If this fails, the plugins that replied 'POSSIBLE' will be tried.
-                   #     NOTE: This last value should be used with great care!
+    """
+    Used internally to determine plugins capable of 'manual' addition of devices, see also [add|remove]ManualDevice below.
+
+    Values:
+     * NO       = 0: The plugin can't add a device 'manually' (or at least not with the given parameters).
+     * POSSIBLE = 1: The plugin will try to add the (specified) device 'manually', unless another plugin has priority.
+     * PRIORITY = 2: The plugin has determined by the specified parameters that it's responsible for adding this device and thus has priority.
+     If this fails, the plugins that replied 'POSSIBLE' will be tried.
+     NOTE: This last value should be used with great care!
+    """
+    NO = 0,
+    POSSIBLE = 1,
+    PRIORITY = 2
```

Comment:
I liked the original formatting better, honestly. This duplicates the entire class in documentation. Since it is quite subjective I'm also fine with leaving it like this though.

Reply:
I did not find any standard in Python on how to document Enums. I'll just revert it to the way it was.